mne/chpi.py

```diff
@@ -314,50 +341,35 @@ def _setup_chpi_fits(info, t_window, t_step_min, method='forward',
     model += [slope, np.ones(slope.shape)]
     model = np.concatenate(model, axis=1)
     inv_model = linalg.pinv(model)
-    # Set up highpass at half lowest cHPI freq
-    hp_n = 2 ** (int(np.ceil(np.log2(n_window))) + 1)
-    freqs = fftpack.rfftfreq(hp_n, 1. / info['sfreq'])
-    hp_ind = np.where(freqs >= hpi_freqs.min())[0][0] - 2
-    hp_window = np.concatenate(
-        [[0], np.repeat(np.hanning(hp_ind - 1)[:(hp_ind - 1) // 2],
-                        2)])[np.newaxis]
```

Comment:
Ahh right. Perhaps I had been using it at one point and then realized it shouldn't be necessary...